droid-hal-device.inc

```diff
@@ -417,7 +417,7 @@ mkdir tmp
 echo Verifying kernel config
 # AOSP seems to use .../obj/kernel/.config not obj/KERNEL_OBJ/.config like CM : so wildcard it
 hybris/mer-kernel-check/mer_verify_kernel_config \
-    %{android_root}/out/target/product/%{device}/obj/*/.config
+    %{android_root}/out/target/product/%{device}/obj/KERNEL_OBJ/.config
```

Comment:
The read the comment above ^ you hard coding the path breaks anything else than anything based on CM.